CHANGELOG

```diff
@@ -0,0 +1,69 @@
+# spark-redshift Changelog
+
+## 4.0.0-SNAPSHOT (2019-06-26)
+
+Baseline SNAPSHOT version working with 2.4
+
+#### Deprecation
+In order to get this baseline snapshot out, we dropped some features and package versions, and disabled some tests.
+Some of these changes are temporary, others - such as dropping hadoop 1.x - are meant to stay.
```

Comment:
You should go farther than this, calling out that our intent is to do the best job possible supporting the minimal set of features that the community needs. Which means that other non-essential features may be dropped before the first non-snapshot release, and that the community's feedback and contributions are vitally important.

Reply:
added